src/OpenLoco/OpenLoco.cpp

```diff
@@ -980,6 +980,9 @@ namespace OpenLoco
                 call(0x004969DA);
                 call(0x00439BA5);
 
+                auto company = CompanyManager::get(CompanyManager::getControllingId());
+                company->updateDaily();
```

Comment:
One of these is probably already company update daily. I think it best we work out which one it is (if it is one)

Reply:
Sure, perhaps in another PR?

Reply:
0x00437FB8 is the function

Reply:
I think i'll quickly implement the function and then you can just add your code in the correct place.

Reply:
https://github.com/OpenLoco/OpenLoco/pull/1288 so you would want to add a method called updateLoanClearance or something like that.

Reply:
Amazing, thanks, I'll update my PR once that's merged back to master

Reply:
Okay can you rebase and rework into the available functions. I created an updateDailyPlayer but you prbably want an updateDailyControllingPlayer or something.